### PR TITLE
Fix: fullDescription field in SARIF output is not correctly escaped

### PR DESCRIPTION
**Pull request by rahul2393**
[Issue](https://github.com/aquasecurity/trivy/issues/604)

### CODE REVIEW

**Comment by simar7** [comment]
lgtm, but this makes me think we should probably escape other fields as well. Like `shortDescription` might be a candidate. What do you think @rahul2393

**Comment by knqyf263** [comment]
I agree with @simar7. `escapeString` doesn't escape `\`, right?

**Comment by rahul2393** [comment]
@knqyf263 yes it does not escape, I am adding to all then

**Comment by rahul2393** [comment]
@knqyf263 @simar7  Done

File: contrib/sarif.tpl [hunk 1]
```diff
@@ -20,20 +20,20 @@
...
+                "text": "Vulnerability {{ .VulnerabilityID }}\nSeverity: {{ .Vulnerability.Severity }}\nPackage: {{ .PkgName | printf "%v" }}\nInstalled Version: {{ .InstalledVersion }}\nFixed Version: {{ .FixedVersion }}\nLink: [{{ .VulnerabilityID }}](https://nvd.nist.gov/vuln/detail/{{ .VulnerabilityID | toLower }})",
+                "markdown": "**Vulnerability {{ .VulnerabilityID }}**\n| Severity | Package | Installed Version | Fixed Version | Link |\n| --- | --- | --- | --- | --- |\n|{{ .Vulnerability.Severity }}|{{ .PkgName | printf "%v" }}|{{ .InstalledVersion }}|{{ .FixedVersion }}|[{{ .VulnerabilityID }}](https://nvd.nist.gov/vuln/detail/{{ .VulnerabilityID | toLower }})|\n"
               },
               "properties": {
                 "tags": [
                   "vulnerability",
                   "{{ .Vulnerability.Severity }}",
-                  "{{ .PkgName }}"
+                  "{{ .PkgName | printf "%v" }}"
```
Comment (simar7) [inline contrib/sarif.tpl]: Can you add a test that exercises these fields as well?

Comment (rahul2393) [reply contrib/sarif.tpl]: @simar7 added test

**Comment by simar7** [comment]
lgtm

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**